FluentUI.Demo/build.gradle

```diff
@@ -85,19 +84,21 @@ dependencies {
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
     implementation 'androidx.cardview:cardview:1.0.0'
     implementation "androidx.constraintlayout:constraintlayout:$constraintLayoutVersion"
-    implementation project(':FluentUI')
     implementation 'com.squareup.picasso:picasso:2.71828'
     implementation 'com.github.bumptech.glide:glide:4.8.0'
     implementation 'com.jakewharton.threetenabp:threetenabp:1.1.0'
 
-    implementation "androidx.compose.ui:ui:$composeVersion"
-    implementation ("androidx.compose.foundation:foundation-layout:$composeVersion")
-    implementation ("androidx.compose.material:material:$composeVersion")
+    //Compose BOM
+    implementation platform("androidx.compose:compose-bom:$composeBomVersion")
+    implementation "androidx.compose.ui:ui"
+    implementation ("androidx.compose.foundation:foundation-layout")
+    implementation ("androidx.compose.material:material")
     implementation ("androidx.activity:activity-compose:$composeActivityVersion")
-    implementation 'androidx.test.ext:junit-ktx:1.1.3'
-    androidTestImplementation("androidx.compose.ui:ui-test-junit4:$composeVersion")
-    debugImplementation("androidx.compose.ui:ui-test-manifest:$composeVersion")
-    implementation "androidx.compose.runtime:runtime-livedata:$composeVersion"
+    implementation 'androidx.test.ext:junit-ktx:1.1.5'
+    androidTestImplementation("androidx.compose.ui:ui-test-junit4:$composeTestVersion")
+    debugImplementation("androidx.compose.ui:ui-test-manifest")
+    debugImplementation "androidx.test:monitor:1.6.1"
```

Comment:
is the version supposed to be hardcoded here?

Reply:
Had to add this to fix some missing modules during test runs, these are present in this location only, and not intended to be used in other modules.
Do you recommend moving them out?

Reply:
lets move the version out